Add walkthrough guide (#310)
HealthScreening/HealthScreeningBot: guides/walkthrough/page_3.md

__**Miscellaneous Commands**__:

> `/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.

Reviewer: ```suggestion

`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
```

> Paremeters: 

Reviewer: ```suggestion
__Parameters__:
```

- **name**: The name of the guide to send.
- **paginate**: Whether to paginate the guide. This is an optional argument and should be either __True__ or __False__. The default is __True__. Note: One-page guides will not have any paginator buttons regardless of the value provided to this argument.
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.

`/profile`: Shows an overview of all information stored for the user, including information submitted via `/set_auto` and `/set`.

> - **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.

Reviewer: ```suggestion
__Parameters__:
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

> `/stats` : Send bot stats. This does not have the ephemeral argument (why would it?)

Reviewer: ```suggestion
`/stats`: Send bot stats. This does not have the ephemeral argument (why would it?)
```

> **Miscellaneous**

Reviewer: ```suggestion
__**Miscellaneous Commands**__:
```

> *Note* All commands in this category have an ephemeral argument, which hides the content sent from everyone else if True. This argument will not be listed in these parameters.

Reviewer: This isn't true for stats.
```suggestion
```

> `guide`: Sends guides from the bot's github.

Reviewer: ```suggestion
`/guide`: Sends a guide from a list of approved guides. Guides can be contributed by anyone to the bot's GitHub repository.
```

Paremeters: 

> - name : Name of the guide to send

Reviewer: ```suggestion
- **name**: The name of the guide to send.
```

> - paginate : Whether to paginate the guide. This is an optional argument.

Reviewer: ```suggestion
- **paginate**: Whether to paginate the guide. This is an optional argument and should be either __True__ or __False__. The default is __True__. Note: One-page guides will not have any paginator buttons regardless of the value provided to this argument.
```

> 

Reviewer: ```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

> `profile` : Sends the information set with the `set` command.

Reviewer: ```suggestion
`/profile`: Shows an overview of all information stored for the user, including information submitted via `/set_auto` and `/set`.
```

> 

Reviewer: ```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

> `stats` : Send bot stats. This does not have the ephemeral argument (why would it?)

Reviewer: ```suggestion
`/stats` : Send bot stats. This does not have the ephemeral argument (why would it?)
```
